src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,59 @@
 package mate.jdbc;
 
+import java.util.ArrayList;
+import java.util.List;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.CarService;
+import mate.jdbc.service.DriverService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        CarService carService = (CarService) injector.getInstance(CarService.class);
+        System.out.println("All cars:");
+        List<Car> cars = carService.getAll();
+        cars.forEach(System.out::println);
+        System.out.println(System.lineSeparator() + "Car with id 2:");
+        Car car = carService.get(2L);
+        System.out.println(car);
+        System.out.println(System.lineSeparator() + "Add new car:");
+        Manufacturer manufacturer = new Manufacturer("Peugeot", "France");
+        manufacturer.setId(25L);
```

Comment:
we don't need to set id, because the method create() in manufacturerServiceImpl should generate a key